workitem/workitem_repository_blackbox_test.go

```diff
@@ -322,3 +322,43 @@ func (s *workItemRepoBlackBoxTest) TestLookupIDByNamedSpaceAndNumberNotFound() {
 	require.NotNil(s.T(), err)
 	assert.IsType(s.T(), errors.NotFoundError{}, errs.Cause(err))
 }
+
+func (s *workItemRepoBlackBoxTest) TestLookupIDByNamedSpaceAndNumberStaleSpace() {
+	testFxt := tf.NewTestFixture(s.T(), s.DB, tf.WorkItems(20, func(testf *tf.TestFixture, idx int) error {
+		testf.WorkItems[idx].Fields[workitem.SystemState] = workitem.SystemStateNew
+		testf.WorkItems[idx].Fields[workitem.SystemCreator] = testf.Identities[0].ID.String()
+		return nil
+	}))
+	sp := *testFxt.Spaces[0]
+	wi := *testFxt.WorkItems[0]
+	in := *testFxt.Identities[0]
+	wiID, spaceID, err := s.repo.LookupIDByNamedSpaceAndNumber(s.Ctx, in.Username, sp.Name, wi.Number)
```

Comment:
The method name has a typo (Named). s/LookupIDByNamedSpaceAndNumber/LookupIDByNameSpaceAndNumber
But this many not be under the scope of this PR.

Reply:
Thanks, @baijum, Not a typo, we have `named`spaces / `named`workItems etc in our codebase.